ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -752,4 +752,16 @@ MAU_US_SBO_1000_1:
   - MAU_SFSS_SFVSC_HTSC_BPC_SHM
   - DEPRECATED
   uses:
-  - humidification_run_command
\ No newline at end of file
+  - humidification_run_command
+  
+ 
+#Canonical for Air washers#
+MAU_SFSS_FDPM_SSPM_STM_SHM:
+  description: "makeup Air Unit with run command, status, dufferential pressure, static T,RH, monitoring and control"
+  implements:
+  - MAU
+  - SFSS
+  - FDPM
+  - SSPM
+  - STM
```

Comment:
How does it do this? DX or CHW?

Reply:
The unit has water spray and doesnot have DX or CHW associated to the unit.